src/widgets/urlobject.cpp

```diff
@@ -189,21 +189,21 @@ QMenu *URLObject::createPopupMenu(const QString &lnk)
     QMenu *m = new QMenu;
 
     bool needGenericOpen = true;
-    if (service == "mailto" || service == "x-psi-atstyle") {
-        needGenericOpen = false;
-        m->addAction(d->act_mailto);
-    }
-    if (service == "jabber" || service == "jid" || service == "xmpp" || service == "x-psi-atstyle") {
```

Comment:
could you elaborate a little about jid/jabber removal? I just don't remember where it comes from and how it works

Reply:
It has been replaced by "xmpp" a long time ago...
- https://github.com/psi-im/neatstuff/blob/master/psiwidgets/psitextview.cpp#L58
- https://github.com/psi-im/psi/blob/master/src/textutil.cpp#L341
- https://github.com/psi-im/psi/blob/master/src/psicon.cpp#L719
- https://github.com/psi-im/psi/blob/master/src/psicon.cpp#L792
- https://github.com/psi-im/psi/blob/master/src/psicon.cpp#L1291